waspc/data/Generator/templates/Dockerfile

```diff
@@ -33,3 +33,7 @@ COPY db/ ./db/
 EXPOSE ${PORT}
 WORKDIR /app/server
 ENTRYPOINT ["npm", "run", "start-production"]
+
+
+# Any user-defined Dockerfile contents will be appended below.
+{=& customDockerfile =}
```

Comment:
We are mixing some names a bit:
1. User-defined dockerfile
2. Custom dockerfile
3. optional dockerfile (in AppSpec)
4. somewhere you call it just dockerfile (ok, in some contexts that is ok).

Maybe we should stick to `userDockerfile`?

Reply:
Good thinking on unifying the naming, will do! 👍🏻 Thnx